### DISCUSSION TITLE
MessageLoggerV2

### DISCUSSION THREAD

**Discussion by Aerestic** [community/community#30669] [Other Feature Feedback, Questions, & Ideas]
MessageLoggerV2 makes my discord disconnect and connect call after I send a message or receive a message.

**Comment by airtower-luna** [comment]
It looks like you're looking for help with a software hosted on GitHub, though I have no idea what. Anyway, in that case you should open an issue *in the repository for that software* so the developers see it. :slightly_smiling_face:

**Comment by JxVtrl** [comment]
same problem
